### PR TITLE
Add Userpilot (Actions) Destination Docs

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Userpilot new integration docs
- Update description
- Add ID to Userpilot Docs

### Merge timing

- ASAP once approved.

### Related issues (optional)

#5000

### CODE REVIEW

File: src/connections/destinations/catalog/actions-userpilot-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+id: 647f30a35eedd03afde0a1c3
+beta: true
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+Userpilot helps product teams deliver personalized in-app experiences to increase growth metrics at every stage of the user journey. When you integrate Userpilot with Segment, you can send your Segment events to Userpilot, which allows you to create more personalized experiences for your users.
+
+
+This destination is maintained by Userpilot. For any issues with the destination, [contact their Support team](mailto:support@userpilot.co).
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-cloud/index.md:13]:
```suggestion
This destination is maintained by Userpilot. For any issues with the destination, [contact Userpilot's Support team](mailto:support@userpilot.co){:target="_blank"}.
```

File: src/connections/destinations/catalog/actions-userpilot-cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{% include components/actions-fields.html %}
+
+
+## Troubleshooting
+
+If you experience any issues while setting up Userpilot as a destination, follow these steps:
+
+- Check your Userpilot API Key. Make sure it's correctly entered in Segment.
+- Verify that you've enabled Userpilot as a destination in Segment.
+- If you're still having trouble, contact Segment's support team for further assistance.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-cloud/index.md:46]:
```suggestion
- If you're still having trouble, [contact Segment's support team](https://segment.com/help/contact/){:target="_blank"} for further assistance.
```

File: src/connections/destinations/catalog/actions-userpilot-web/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Userpilot helps product teams deliver personalized in-app experiences to increase growth metrics at every stage of the user journey. When you integrate Userpilot with Segment, you can send your Segment events to Userpilot, which allows you to create more personalized experiences for your users.
+
+
+This destination is maintained by Userpilot. For any issues with the destination, [contact their Support team](mailto:support@userpilot.co).
+
+{% include content/ajs-upgrade.md %}
+
+
+## Getting Started
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-web/index.md:18]:
```suggestion
## Getting started
```

File: src/connections/destinations/catalog/actions-userpilot-web/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+id: 6480b4eeab29eca5415089d4
+beta: true
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+Userpilot helps product teams deliver personalized in-app experiences to increase growth metrics at every stage of the user journey. When you integrate Userpilot with Segment, you can send your Segment events to Userpilot, which allows you to create more personalized experiences for your users.
+
+
+This destination is maintained by Userpilot. For any issues with the destination, [contact their Support team](mailto:support@userpilot.co).
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-web/index.md:13]:
```suggestion
This destination is maintained by Userpilot. For any issues with the destination, [contact Userpilot's Support team](mailto:support@userpilot.co){:target="_blank"}.
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]

**Review by rchinn-segment — APPROVED** [review]

File: src/connections/destinations/catalog/actions-userpilot-cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+Userpilot helps product teams deliver personalized in-app experiences to increase growth metrics at every stage of the user journey. When you integrate Userpilot with Segment, you can send your Segment events to Userpilot, which allows you to create more personalized experiences for your users.
+
+
+This destination is maintained by Userpilot. For any issues with the destination, [contact Userpilot's Support team](mailto:support@userpilot.co){:target="_blank"}.
+
+{% include content/ajs-upgrade.md %}
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-userpilot-cloud/index.md:15]: You can remove this snippet as it is only relevant for the web destination
```suggestion
```

**Comment by tcgilbert** [comment]
@rchinn-segment can we get this one merged before site rebuild tomorrow?

File: src/connections/destinations/catalog/actions-userpilot-cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Userpilot Cloud (Actions)**.
+4. Select an existing Source to connect to Userpilot Cloud (Actions).
+5. Find your Userpilot API key and API endpoint in the [environment dashboard](https://run.userpilot.io/environment){:target="_blank"}.
+
+### Overview
+
+The Userpilot cloud-mode destination uses [Userpilot’s REST APIs](https://docs.userpilot.com/article/195-identify-users-and-track-api){:target="_blank"} to transmit user data and associated events directly to Userpilot. This lets you use Userpilot’s capabilities based on the real-time data received from your application.
+
+- **User Identification** Send [Identify](/docs/connections/spec/identify/) calls from Segment to Userpilot for identifying or updating user and company properties. This data is dispatched directly from your backend servers and can be used for segmenting users and triggering personalized content in real-time
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-cloud/index.md:28]:
```suggestion
- **User Identification** Send [Identify](/docs/connections/spec/identify/) calls from Segment to Userpilot for identifying or updating user and company properties. This data is dispatched directly from your backend servers and can be used for segmenting users and triggering personalized content in real-time.
```

File: src/connections/destinations/catalog/actions-userpilot-web/index.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+If you're not familiar with the Segment Specs, take a look to understand what the [Identify method](/docs/connections/spec/identify/) does. An example call would look like:
+
+```js
+analytics.identify('userId123', {
+  email: 'john.doe@example.com'
+});
+```
+
+Calling `identify` from `analytics.js` will trigger the `userpilot.identify`. We recommend passing as much data as possible to get the most out of Userpilot.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-userpilot-web/index.md:46]:
```suggestion
Calling `identify` from `analytics.js` will trigger the `userpilot.identify`. Segment recommends passing as much data as possible to get the most out of Userpilot.
```

**Review by rchinn-segment — APPROVED** [review]

**Comment by rchinn-segment** [comment]
@tcgilbert - absolutely, we'll merge and include this in our deploy tomorrow

**Pull request merged**